docs/user-view/configuration/staff.md

```diff
@@ -0,0 +1,18 @@
+Currently, our application lacks a user-friendly interface for comfortably managing configurations. Instead, these settings are stored in a JSON file, which requires manual editing to make any changes.
```

Comment:
Shouldn't this be part of the index? This is a general information not specific to the employees.

Reply:
Or maybe put it inside an info box like https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types